.changeset/hungry-sloths-prove.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sumup-oss/circuit-ui': patch
+---
+
+Improved the DateInput component's calendar dialog to be placed relative to the calendar button instead of the whole file. This improves the placement when the DateInput is rendered near the center of the viewport.
```

Comment:
```suggestion
Improved the DateInput component's calendar dialog to be placed relative to the calendar button instead of the whole field. This improves the placement when the DateInput is rendered near the center of the viewport.
```

Reply:
this should also mention the new `placement` prop